ci_tools/prepare_linux.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
```

Comment:
Can we do that in a later commit. These are just for travis and the ci environment.